glyphs-reader/src/font.rs

```diff
@@ -65,6 +62,15 @@ pub struct Font {
     // master id => { (name or class, name or class) => adjustment }
     pub kerning_ltr: Kerning,
 
+    pub custom_parameters: CustomFontParameters,
```

Comment:
how about naming them "FontCustomParameters" instead? To my non-native Engish speaking mind, it reads slightly better as it makes it easier to compare/contrast to the other masters' and instances' custom parameters.
By the way, don't we need to also group the latter two?

Reply:
name sounds good.

And yes, the goal is also to group the other ones, I can take a look at that too.